File: Jour7/jour7.pas
program Maximun;
var
    nbre, max   : real ;
	k                : integer ;   
	egaliter       : integer ;

begin
	k := 0;
	egaliter := 1;
	writeln(' ============== Ce programme trouve le plus grand nombre parmis trois nombres donnees. ============== ');
	repeat
		write('Donner un nombre : ');
		read(nbre);
		if k = 0 then
            max := nbre;
		if k > 0 then
		begin
			if max <> nbre then
				egaliter := 0
		end;
		if nbre > max then
			max := nbre;
		k += 1;
	until(k >= 3);

	if egaliter = 1 then
	begin
		writeln('Les trois nombres fournis sont egaux et valent : ',max:0:3,'.');
	end
	else
	begin
		writeln('Les trois nombres fournis differts et le plus grand est : ',max:0:3,'.');
	end;
end.

File: Jour7/jour7.c
#include<stdio.h>
#include<stdbool.h>

int main()
{
	float nbre, max;
	int k=0;
	bool egaliter;
	printf(" ============== Ce programme trouve le plus grand nombre parmis trois nombres donnees. ==============\n");
	do
	{
		printf("Donner un nombre : ");
		scanf("%f",&nbre);
		if( k == 0 )
			max = nbre;
		if (k > 0)
			if (max != nbre)
				egaliter = false;

		if(nbre > max)
			max = nbre;
		k += 1;
	}while(k < 3);

	if (egaliter)
		printf("Les trois nombres fournis sont egaux et valent : %f.",max);
	else
		printf("Les trois nombres fournis differts et le plus grand est : %f.",max);

	return 0;
}